scripts/pi-hole/php/savesettings.php

```diff
@@ -376,16 +378,8 @@ function addStaticDHCPLease($mac, $ip, $hostname) {
 				{
 					$IPs = implode (",", $DNSservers);
 					$return = pihole_execute("-a setdns \"".$IPs."\" ".$extra);
-					if(!empty($return))
-					{
-						$success .= htmlspecialchars(end($return))."<br>";
-						$success .= "The DNS settings have been updated (using ".$DNSservercount." DNS servers)";
-					}
-					else
-					{
-						$success .= "Updating DNS settings failed. Result:";
-						$success .= implode($return);
-					}
+					$success .= htmlspecialchars(end($return))."<br>";
```

Comment:
So we always assume success now (if the checks above worked)?

Reply:
Well... this is how it was before the `pihole` wrapper code was put in.